conftest.py

```diff
@@ -0,0 +1,3 @@
+import sys
+
+sys.path.append("micropip")
```

Comment:
Are you sure about this? Normally it shouldn't be necessary.

Reply:
I'm not sure why but that was needed for me to run test locally. I'll remove that file and let's see if CI complains.

Reply:
Okay, it works. Maybe my local setting was wrong.
(The test failure is about codecov upload)

Reply:
> (The test failure is about codecov upload)

This happens a lot, it's a bit annoying. If it's going to happen regularly, maybe we can figure out a way to not fail the test when the upload fails.